<launch>
    <!-- STATIC TF PUBLISHER NODES -->
    <node pkg="tf" type="static_transform_publisher" name="base_broadcaster" args="0 0 0 0 0 0 1 world comau_smart_six/base_link 10" />
    <!-- COMAU IK SERVICES -->
    <include file="$(find rocup)launch/robots/comau/comau_real_stack.launch" />
    <!-- World Frame tf ID Name -->
    <param name="WORLD_FRAME_ID" value="world"/>
    <!-- Robot -->
    <arg name="robot_name" default="comau_smart_six"/>
    <!--Robot ID Names -->
    <param name="COMAU_NAME" value="$(arg robot_name)"/>
    <!-- Trajectories Time -->
    <param name="$(arg robot_name)/TRAJECTORY_TIME" value="10"/>
    <!-- Nodes Frequency  -->
    <param name="$(arg robot_name)/NODE_FREQUENCY" value="300"/>
    <!-- MONGODB -->
    <include file="$(find mongodb_store)/launch/mongodb_store.launch">
        <arg name="db_path" value="/opt/ros/mongodb_store/" />
    </include>
    <!-- JOYSTICK -->
    <node name="joy_node" pkg="joy" type="joy_node" output="screen"/>
    <!-- TF -->
    <node name="tf_manager" pkg="rocup" type="tf_manager.py" output="screen" />
    <node name="tf_filter_supervisor" pkg="rocup" type="tf_filter.py" output="screen" />
</launch>